client/src/pages/SignUp/index.tsx

```diff
@@ -1,18 +1,27 @@
 import { FC } from 'react';
 
-import { StyledLoginPage } from './index.style';
+import { StyledSignUpPage } from './index.style';
 import LeftDoodles from '~/components/signup/LeftDoodles';
 import RightDoodles from '~/components/signup/RightDoodles';
+import SignUpTypes from '~/components/signup/SignUpTypes';
 import SignUpForm from '~/components/signup/SignUpForm';
-// import SignUpTypes from '~/components/signup/SignUpTypes';
+import { useParams } from '~/core/Router';
 
+/**
+ * /signup/:stage
+ * stage: one of ['select', 'own', 'facebook', 'google']
+ */
 const SignUpPage: FC = () => {
+  const { stage } = useParams();
   return (
-    <StyledLoginPage>
+    <StyledSignUpPage>
       <LeftDoodles />
-      <SignUpForm />
+      {stage === 'select' ? <SignUpTypes /> : null}
+      {stage === 'own' ? <SignUpForm /> : null}
+      {stage === 'google' ? <SignUpForm social="GOOGLE" /> : null}
+      {stage === 'facebook' ? <SignUpForm social="FACEBOOK" /> : null}
```

Comment:
&&로 바꿨을 수도 있을 것 같아요~

```suggestion
      {stage === 'select' && <SignUpTypes />}
      {stage === 'own' && <SignUpForm />}
      {stage === 'google' && <SignUpForm social="GOOGLE" />}
      {stage === 'facebook' && <SignUpForm social="FACEBOOK" /> }
```

Reply:
저는 null이 편합니다. (단호)